aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -577,42 +863,42 @@ def export_tree(
                 export_data[current_entity].update({
                     temp_d[key]['id']:
                     serialize_dict(
-                        temp_d[key], remove_fields=['id'], rename_fields=model_fields_to_file_fields[current_entity]
+                        temp_d[key],
+                        remove_fields=['id'],
+                        rename_fields=model_fields_to_file_fields[current_entity],
                     )
                 })
 
-    # Close progress up until this point in order to print properly
-    close_progress_bar(leave=False)
+    return export_data
 
-    #######################################
-    # Manually manage attributes and extras
-    #######################################
-    # Pointer. Renaming, since Nodes have now technically been retrieved and "stored"
-    all_node_pks = node_ids_to_be_exported
 
-    model_data = sum(len(model_data) for model_data in export_data.values())
-    if not model_data:
-        EXPORT_LOGGER.log(msg='Nothing to store, exiting...', level=LOG_LEVEL_REPORT)
-        return
-    EXPORT_LOGGER.log(
-        msg=f'Exporting a total of {model_data} database entries, of which {len(all_node_pks)} are Nodes.',
-        level=LOG_LEVEL_REPORT
-    )
+def _get_node_data(all_node_pks: Set[int], silent: bool) -> Iterable[Tuple[str, dict, dict]]:
```

Comment:
```suggestion
def _collect_node_data(all_node_pks: Set[int], silent: bool) -> Iterable[Tuple[str, dict, dict]]:
```

Reply:
fixed in b689f5e6986c15ab18ad225e5659f1ef3bf8f9c0